packaging_automation/citus_package.py

```diff
@@ -264,31 +284,38 @@ def get_docker_image_name(platform: str):
 @validate_parameters
 # disabled since this is related to parameter_validations library methods
 # pylint: disable=no-value-for-parameter
+# pylint: disable= too-many-locals
```

Comment:
I think limiting the number of local variables to 15 is a good practice. How about splitting this into smaller functions and not adding an exception to pylint?

Reply:
The method is 33 lines long which seems very reasonable to me. Splitting the method is not a good option which would hurt the readability I think. If it is required to fit into 15 local variable for here, I can use single tuple values for decode_os_and_release and get_postgres_versions. However, I need to access values with numerical indexes which will make it hard to read